<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVLogFileDialog</class>
 <widget class="QDialog" name="SVLogFileDialog">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>480</width>
	<height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
	<widget class="QLabel" name="labelOpenFileError">
	 <property name="wordWrap">
	  <bool>true</bool>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="SVLogWidget" name="logWidget" native="true"/>
   </item>
   <item>
	<widget class="QDialogButtonBox" name="buttonBox">
	 <property name="orientation">
	  <enum>Qt::Horizontal</enum>
	 </property>
	 <property name="standardButtons">
	  <set>QDialogButtonBox::Ok</set>
	 </property>
	</widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SVLogWidget</class>
   <extends>QWidget</extends>
   <header>SVLogWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SVLogFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>248</x>
	 <y>254</y>
	</hint>
	<hint type="destinationlabel">
	 <x>157</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SVLogFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>316</x>
	 <y>260</y>
	</hint>
	<hint type="destinationlabel">
	 <x>286</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
 </connections>
</ui>
